public/app/features/alerting/unified/components/import-to-gma/ConfirmConvertModal.tsx

```diff
@@ -51,6 +51,25 @@ const AlertSomeRulesSkipped = () => {
   );
 };
 
+const WarningForNotUsingRulesManagedByIntegrations = () => {
```

Comment:
nit; the variable name makes it sounds like we warn users when they _aren't_ importing rules managed by inegrations :D 

Maybe `WarningForImportingRulesManagedByIntegrations`?